contribs/gmf/examples/https.js

```diff
@@ -0,0 +1,9 @@
+/*
+ * Redirect to https
+ */
+if (window.location.protocol == 'http:' &&
+    window.location.hostname != 'localhost') {
+  var restOfUrl = window.location.href.substr(5);
+  /** @type {string|WindowLocation} */
```

Comment:
Oh, I just got the error when running `make check`.
However, we can use the `Location` type. It's provided by the compiler.
```javascript
/** @type {Location} */
```
See
https://github.com/google/closure-compiler/blob/master/externs/browser/window.js#L56